runtime/doc/api.txt

```diff
@@ -706,7 +706,7 @@ nvim_create_buf({listed}, {scratch})                       *nvim_create_buf()*
                     buf_open_scratch
 
 nvim_create_namespace({name})                        *nvim_create_namespace()*
-                Creates a new namespace, or gets an existing one.
+                Creates a new namespace , or gets an existing one.
```

Comment:
```suggestion
                Creates a new namespace, or gets an existing one.
```

Reply:
this is because the source needs to be escaped. i will fix this in the next docs pr